### PR TITLE
feat: 10550 Re-introduce test for measuring cold app start + JS bundle load time

### PR DESCRIPTION
**Pull request by MarioAslau**
## **Description**

Using react-native-performance as a performance measurement tool, which will be implemented here - https://github.com/MetaMask/metamask-mobile/issues/10521, we want to utilize E2E world for measuring cold app start + JS bundle load time.

Technical Details
* Add E2E test that measures The new App Start time in ColdStartToOnboardingScreen.feature replacing the old test
* Added a baseline measure to compare App Start time against.
  Values from 10 tests:
  //test 1: 383
  //test 2: 334
  //test 3: 380
  //test 4: 410
  //test 5: 394
  //test 6: 352
  //test 7: 602
  //test 8: 364
  //test 9: 359
  //test 10: 438
* E2E test check the performance numbers against the baseline value. If the new performance numbers is higher than the threshold, fail the test and vice versa

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by cortisiko — DISMISSED** [review]
🚀 🚀 🚀

**Review by Cal-L — COMMENTED** [review]
Fix lint issues

File: app/components/Views/OnboardingCarousel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+        >
+          <View
+            style={styles.wrapper}
+            testID={OnboardingCarouselSelectorIDs.CAROUSEL_CONTAINER_ID}
+          >
+            <ScrollableTabView
+              renderTabBar={renderTabBar}
+              onChangeTab={onChangeTab}
+            >
+              {['one', 'two', 'three'].map((value, index) => {
```
Comment (tommasini) [inline app/components/Views/OnboardingCarousel/index.tsx:227]: NIT: Should 'one', 'two', 'three' and 1,2,3 be constants?

**Review by tommasini — APPROVED** [review]
LGTM! Awesome!

File: app/components/Views/OnboardingCarousel/index.tsx (hunk 1 above)
Comment (MarioAslau) [reply app/components/Views/OnboardingCarousel/index.tsx:227]: value is being used in testID={`carousel-${value}-image`}

**Pull request merged**